docs/guides/about/glossary.md

```diff
@@ -681,8 +696,8 @@ saturation, add small amounts of its complement, making it closer to grey.
 {#scene}
 ## Scene
 
-A defined portion of the continuous strips of data collected by satellites. Satellite data is broken up into scenes 
-for ease in handling and cataloguing.
+Each [swath](#swath) is divided into a series of scenes to enable the data to be handled and catalogued more easily.
+Therefore, a scene is a defined portion of the continuous strips of data collected by satellites.
```

Comment:
Another caveat is that "scenes" are primarily a Landsat thing - Sentinel-2 divides its data even further into smaller "granules"